apps/admin/frontend/src/screens/contest_adjudication_screen.tsx

```diff
@@ -0,0 +1,1124 @@
+  const queryClient = useQueryClient();
+  const apiClient = useApiClient();
+  useEffect(() => {
+    if (!writeInCvrQueueQuery.isSuccess || !scrollStateInitialized) return;
+    function prefetch(cvrId: Id) {
+      void queryClient.prefetchQuery({
+        queryKey: getCvrWriteInImageViews.queryKey({ cvrId, contestId }),
+        queryFn: () =>
+          apiClient.getCvrContestWriteInImageViews({ cvrId, contestId }),
+      });
+    }
+    const nextCvrId = writeInCvrQueueQuery.data[scrollIndex + 1];
+    if (nextCvrId) {
+      prefetch(nextCvrId);
+    }
+    const prevCvrId = writeInCvrQueueQuery.data[scrollIndex - 1];
+    if (prevCvrId) {
+      prefetch(prevCvrId);
+    }
+  }, [
+    apiClient,
+    contestId,
+    queryClient,
+    scrollIndex,
+    scrollStateInitialized,
+    writeInCvrQueueQuery,
+  ]);
+
+  const areQueriesLoading =
+    !firstPendingWriteInCvrIdQuery.isSuccess ||
+    !writeInCvrQueueQuery.isSuccess ||
+    !cvrVoteInfoQuery.isSuccess ||
+    !cvrContestWriteInImagesQuery.isSuccess ||
+    !cvrContestWriteInsQuery.isSuccess ||
+    !contestWriteInCandidatesQuery.isSuccess ||
+    !cvrContestVoteAdjudicationsQuery.isSuccess;
+
+  if (
+    !(scrollStateInitialized && voteStateInitialized) ||
+    (areQueriesLoading && !isStateStale)
+  ) {
+    return (
+      <NavigationScreen title="Contest Adjudication">
+        <Loading isFullscreen />
+      </NavigationScreen>
+    );
+  }
+
+  return (
+    <Screen>
+      <Main flexRow>
+        <BallotPanel>
+          {isFocusedWriteInHmpb && (
+            <BallotZoomImageViewer
+              ballotBounds={firstWriteInImage.ballotCoordinates}
+              key={currentCvrId} // Reset zoom state for each write-in
+              imageUrl={firstWriteInImage.imageUrl}
+              zoomedInBounds={
+                focusedWriteInImage &&
+                'writeInCoordinates' in focusedWriteInImage
+                  ? focusedWriteInImage.writeInCoordinates
+                  : firstWriteInImage.contestCoordinates
+              }
+            />
+          )}
+          {isFocusedWriteInBmd && (
+            <BallotStaticImageViewer
+              key={currentCvrId}
+              imageUrl={firstWriteInImage.imageUrl}
+            />
+          )}
+        </BallotPanel>
+        <AdjudicationPanel>
+          {focusedOptionId && <AdjudicationPanelOverlay />}
+          <BallotHeader>
+            <ContestTitleDiv>
+              <StyledH2>{getContestDistrictName(election, contest)}</StyledH2>
+              <StyledH1>{contest.title}</StyledH1>
+            </ContestTitleDiv>
+            <LinkButton
+              fill="outlined"
+              icon="X"
+              to={routerPaths.writeIns}
+              variant="inverseNeutral"
+            >
+              Close
+            </LinkButton>
+          </BallotHeader>
+          <BallotVoteCount>
+            <MediumText>
+              Votes cast: {voteCount} of {seatCount}
```

Comment:
Along with `import { format } from '@votingworks/utils';` above, consider using our count formatting to set a good example even though it won't ever matter here.

```suggestion
              Votes cast: {format.count(voteCount)} of {format.count(seatCount)}
```